src/test/java/org/lmdbjava/DbiTest.java

```diff
@@ -146,6 +147,22 @@ public void drop() {
   }
 
   @Test
+  public void dropAndDelete() {
+    final Dbi<ByteBuffer> db = env.openDbi(DB_1, MDB_CREATE);
+    final Dbi<ByteBuffer> nameDb = env.openDbi((byte[]) null);
```

Comment:
Would you mind just adding a quick extra test for `null` database, just so we can see it doesn't segmentation fault on any of the supported platforms (the CI server will confirm this)?